src/NHibernate.Test/LazyProperty/LazyPropertyFixture.cs

```diff
@@ -50,7 +51,7 @@ protected override void OnTearDown()
 			using (var s = OpenSession())
 			using (var tx = s.BeginTransaction())
 			{
-				Assert.That(s.CreateSQLQuery("delete from Book").ExecuteUpdate(), Is.EqualTo(1));
+				s.CreateQuery("delete from Book").ExecuteUpdate();
```

Comment:
This assert in teardown is indeed "completing" the `CanLoadAndSaveObjectInDifferentSessions`. Removed and `CanLoadAndSaveObjectInDifferentSessions` test changed for correctly asserting itself all that it needs to assert.